.github/workflows/ci.yml

```diff
@@ -31,7 +31,32 @@ jobs:
       run: dotnet restore
 
     - name: Build
-      run: dotnet build --configuration Release --no-restore
+      run: dotnet build --configuration Release --no-restore --property:ExposeExperimentalFeatures=false
+
+    - name: Test ${{ matrix.version }}
+      run: dotnet test **/bin/**/${{ matrix.version }}/*.Tests.dll --logger:"console;verbosity=detailed"
+
+  build-test-experimental:
+    strategy:
+      fail-fast: false  # ensures the entire test matrix is run, even if one permutation fails
+      matrix:
+        os: [ windows-latest, ubuntu-latest ]
+        version: [ net462, net6.0, net7.0 ]
+        exclude:
+        - os: ubuntu-latest
+          version: net462
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # fetching all
+
+    - name: Install dependencies
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build --configuration Release --no-restore --property:ExposeExperimentalFeatures=true
```

Comment:
Do we want to update the ApiCompatibility and AOT tests to also run for the experimental features as they are added? Or do you think it's okay to have that tested when they get upgraded from an experimental feature to a regular feature?

Reply:
The current default for `ExposeExperimentalFeatures` when unset is `true` so it was actually behaving that way but I updated the workflows to explicitly set the value so the intention is clear.